language-support/hs/bindings/examples/nim/package.yaml

```diff
@@ -0,0 +1,37 @@
+name:                nim
+version:             0.1.0.0
+description:         Play Nim on a DAML Ledger
+license:             Apache-2.0
+author:              The DAML Authors
+maintainer:          nick.chapman@digitalasset.com
+github:              https://github.com/digital-asset/daml
+
+dependencies:
+- ansi-terminal
+- base >= 4.7 && < 5
+- containers
+- daml-ledger
+- extra
+- haskeline
+- random
+- text
+- transformers
+- uuid
+
+executables:
+  nim-exe:
+    main:                DA.Ledger.App.Nim.Main
+    source-dirs:         src
+    ghc-options:
+    - -Wall
+    - -Wno-name-shadowing
+    - -threaded
+    - -rtsopts
+    - -with-rtsopts=-N
+
+    - -XGeneralizedNewtypeDeriving
```

Comment:
`hpack` (and `.cabal` files) have a `default-extensions` field that is specifically meant for this and should be preferred.

Reply:
Did this.